### PR TITLE
recording kubenurse queries to speed up the timing out alerts

### PR DESCRIPTION
**Pull request by myaser**
teapot/issues#1950

### CODE REVIEW

**Comment by mikkeloscar** [comment]
:+1:

**Comment by myaser** [comment]
👍🏻

File: cluster/manifests/prometheus/configmap.yaml [hunk 1]
```diff
@@ -53,7 +53,22 @@ data:
       rules:
       - record: job:skipper_count_by_host_code:sum
         expr: sum(rate(skipper_serve_host_count{}[1m])) by (host,code)
+    {{if eq .Cluster.ConfigItems.network_monitoring_enabled "true" }}
```
Comment (linki) [inline cluster/manifests/prometheus/configmap.yaml:56]: nit: it's easier to see the `if` when it's not indented, similar to https://github.com/zalando-incubator/kubernetes-on-aws/blob/ddcf978e9dfb27851d13f3727adf1264f193ab08/cluster/manifests/prometheus/configmap.yaml#L82-L93

**Comment by myaser** [comment]
👍🏻

**Comment by linki** [comment]
👍

**Pull request merged**